docs/components/InfoPair.js

```diff
@@ -2,10 +2,11 @@ export default {
   header: 'Info Pair',
   description: 'This is a customizable info pair component.',
   snippet:
-        `<ao-info-pair :label="label">
+        `<ao-info-pair :label="label" :label-html="labelHtml">
   {{ info }}
 </ao-info-pair>`,
   apiRows: [
-    { name: 'label', type: 'String, required', default: '-', description: 'Label for info pair.' }
+    { name: 'label', type: 'String', default: '-', description: 'Label for info pair.' },
```

Comment:
lol when was the component's default string for label ever -

Reply:
probably means nothing actually

Reply:
this is a feature across app. way to convey empty string.